src/index.js

```diff
@@ -63,24 +57,34 @@ const updateCheck = (list) => {
   });
 };
 
+const taskDefaul = () => {
+  tasksList.push(new Task('Read', false, (ids += 1)));
+  tasksList.push(new Task('Clean', false, (ids += 1)));
+  tasksList.push(new Task('Run', false, (ids += 1)));
+};
```

Comment:
- [ ] On this part of the project is not required to have some tasks as an example, please remove it. 🐶